raven/base.py

```diff
@@ -627,7 +631,9 @@ def capture(self, event_type, data=None, date=None, time_spent=None,
             event_type, data, date, time_spent, extra, stack, tags=tags,
             **kwargs)
 
-        self.send(**data)
+        # should this event be sampled?
+        if self._random.random() <= self.sample_rate:
```

Comment:
Technically [this is lower bound inclusive](https://docs.python.org/2/library/random.html#random.random), so a sample rate of `0.0` can still result in an event being sent. It'd be safer/more correct to make this comparison `<`. :nerd_face: